Map.h

```diff
@@ -40,6 +40,8 @@ class Map {
     std::vector<Train*> trains;
     std::vector<Station*> stations;
     std::vector<railWay*> railSprites;
```

Comment:
почему в названии этой переменной есть слово sprite, а в предыдущих - нет?

```suggestion
    std::vector<railWay*> railways;
```

Reply:
потому что класс только для отрисовки

Reply:
а классы Train и Station не только для отрисовки?